tests/mmm/test_base.py

```diff
@@ -30,6 +33,123 @@
 )
 
 
+@pytest.fixture(scope="module")
+def plotting_mmm():
+    class ToyMMM(MMM):
+        def build_model(self, data_df, **kwargs):
+            pass
+
+    mmm = ToyMMM(
+        toy_df,
+        target_column="y",
+        date_column="date",
+        channel_columns=["channel_1", "channel_2"],
+    )
+    mmm.control_columns = ["control_1", "control_2"]
```

Comment:
Can we also test when there are no control variables ?

Reply:
Done